docs/src/pages/components/slider-styled/ContinuousSlider.js

```diff
@@ -1,27 +1,24 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { customStyled } from '@material-ui/core/styles';
```

Comment:
Regarding the naming question:

- It raises the question of what we will do with the modules of `@material-ui/styles`.
- It raises the question of why the styles folder behavior differently to the component folder?
Why don't we have `import makeStyles from '@material-ui/core/makeStyles';`?

Maybe?

- Have

```suggestion
import styled from '@material-ui/core/styled';
```

- Treat `'@material-ui/core/styles'` as legacy, rename it later on to `/legacy_styles`.
- Rename 

```diff
-import { styled } from '@material-ui/core';
+import { legacy_styled } from '@material-ui/core';
```

Reply:
`experimentalStyled`?

Reply:
I like using the new utility like this:
```
import styled from '@material-ui/core/styled';
```
That way we can rename the previous one as `legacy_styled`. Also we could more easily document what are the changes, as the imports will come from different places.

@eps1lon do you agree with this too?

Reply:
Or should we export the `customStyled` as `experimentalStyled` and deal with the deprecation/legacy once we have solution for the system?

Reply:
My problem is with the "custom" prefix. It doesn't describe what it is. I don't have a opinion whether the old one is called "legacy" and the new one becomes current or if we call the old one current and the new one "experimental".

Reply:
Got ya, let's rename then the old one to legacy and the new one to simply `styled`.

Reply:
The renaming is done.